src/Coreutils-EYAD_HUSSEIN/pkg/env.go

```diff
@@ -0,0 +1,15 @@
+package pkg
+
+import (
+	"fmt"
+	"os"
+	"strings"
+)
+
+func Env() {
+
+	for _, env := range os.Environ() {
+		envPair := strings.Split(env, "=")
+		fmt.Println(envPair[0], envPair[1])
```

Comment:
no need to split, just print env as it is

Reply:
Done